src/app/products/clothing.ts

```diff
@@ -0,0 +1,190 @@
+    percentCanadian: 100,
+    notes: "A sustainable basics brand making high-quality, minimalist clothing in Canada.",
+    categoryId: "clothing",
+    aliases: ["franc apparel", "franc clothing"],
+    popularity: 83
+    },
+    {
+    name: "Kotn",
+    company: "Kotn",
+    percentCanadian: 100,
+    notes: "A sustainable clothing brand known for ethically sourced cotton and timeless basics.",
+    categoryId: "clothing",
+    aliases: ["kotn clothing", "kotn apparel"],
+    popularity: 89
+    },
+    {
+    name: "Horses Atelier",
+    company: "Horses Atelier",
+    percentCanadian: 100,
+    notes: "A Toronto-based womenswear brand specializing in high-quality, ethically made garments.",
+    categoryId: "clothing",
+    aliases: ["horses atelier clothing", "horses atelier fashion"],
+    popularity: 74
+    },
+    {
+    name: "Province of Canada",
+    company: "Province of Canada",
+    percentCanadian: 100,
+    notes: "A lifestyle brand producing premium Canadian-made clothing inspired by Canadian heritage.",
+    categoryId: "clothing",
+    aliases: ["province of canada apparel", "province of canada clothing"],
+    popularity: 81
+    },
+    {
+    name: "Comrags",
+    company: "Comrags",
+    percentCanadian: 100,
+    notes: "A Toronto-based womenswear brand known for stylish, locally made designs.",
+    categoryId: "clothing",
+    aliases: ["comrags clothing", "comrags fashion"],
+    popularity: 66
+  },
+  {
+    name: 'Joe Fresh',
+    company: 'Joe Fresh',
+    percentCanadian: 80,
+    notes: 'A longstanding Candian staple, featured in Loblaws stores. Although Joe Fresh is a Canadian brand, its products are manufactured overseas, and it has expanded significantly into the U.S. market.',
+    categoryId: 'clothing',
+    aliases: ['joe fresh clothing', 'joe fresh apparel'],
+    popularity: 85
+  },
+  {
+    name: 'Winners',
+    company: 'TJX Companies',
+    percentCanadian: 0,
+    notes: 'Winners is a Canadian discount retail chain owned by the U.S.-based TJX Companies, offering a variety of clothing brands, many of which are not Canadian.',
+    categoryId: 'clothing',
+    aliases: ['winners clothing', 'winners apparel'],
+    popularity: 90
+  },
+  {
+    name: 'American Tall',
+    company: 'American Tall',
+    percentCanadian: 80,
+    notes: 'American Tall is a multigenerational, Canadian business, based in Ontario. As the company expanded, it began manufacturing most of it\'s products in Asia',
+    categoryId: 'clothing',
+    aliases: ['american tall clothing', 'american tall apparel'],
+    popularity: 70
+  },
+  {
+    name: 'American Tall',
+    company: 'American Tall',
+    percentCanadian: 80,
+    notes: 'American Tall is a multigenerational, Canadian business, based in Ontario. As the company expanded, it began manufacturing most of it\'s products in Asia',
+    categoryId: 'clothing',
+    aliases: ['american tall clothing', 'american tall apparel'],
+    popularity: 70
+  },
```

Comment:
```suggestion

```
Looks like we have a dupe here with the item above! (We just have a bug in the code where things can blow up if 2 matching items exist)